code/game/jobs/job/science_vr.dm

```diff
@@ -11,4 +11,11 @@
 			            access_RC_announce, access_keycard_auth, access_tcomsat, access_gateway, access_xenoarch, access_eva, access_network)
 
 /datum/job/scientist
-	alt_titles = list("Xenoarcheologist", "Anomalist", "Phoron Researcher", "Circuit Designer")
\ No newline at end of file
+	spawn_positions = 5
```

Comment:
This feels like a bit much. What is the default? 4 should be plenty. Nice even number.

Reply:
That's the roundstart values, not the total. It makes roundstart match max job slots, iirc.

Reply:
We already have 3/5 scienstist values. This will just bring the roundstart allowed amount to match (same as majority of the jobs)